randovania/games/dread/json_data/Dairon.txt

```diff
@@ -405,16 +405,16 @@ Extra - asset_id: collision_camera_005
   * Extra - actor_def: actordef:actors/props/dooremmy/charclasses/dooremmy.bmsad
   > Door from Yellow EMMI Introduction
       Trivial
-  > Tile Group (SPEEDBOOST)
-      Trivial
+  > Event - Ferenia Transport Speed Blocks Destroyed
+      Speed Booster
```

Comment:
"Dock to Hub Access" is missing a direct connection to "Door to East Transport to Ferenia". Yes destroying the speedboost blocks will take you to the door, but if logic enters this room again with the speedboost blocks destroyed, then it will think "Dock to Hub Access" has no way to access "Door to East Transport to Ferenia".

Reply:
Does this actually matter? I thought there was something a bit ago that allows for revisiting events